### DISCUSSION TITLE
Описание сборки

### DISCUSSION THREAD

**Discussion by 6opogator** [petrovvlad/freeLib#123] [General]
1. В описании сборки забыли указать пакеты qt6-httpserver-dev qt6-websockets-dev
2. Для запуска пакет libqt6sql6-sqlite
Поправьте README.md, пожалуйста. И спасибо за отличную программу.

**Comment by petrovvlad** [comment]
В Ubuntu 22.04 к сожалению не входит qt6-httpserver-dev. В какой ОС вы собирали?

**Reply by 6opogator** [reply]
Debian bookworm
